src/Interpreters/QueryThreadLog.cpp

```diff
@@ -1,70 +1,69 @@
 #include <Interpreters/ProfileEventsExt.h>
-#include <Common/ClickHouseRevision.h>
+#include <Interpreters/QueryLog.h>
 #include <Poco/Net/IPAddress.h>
-#include <array>
+#include <Common/ClickHouseRevision.h>
 
 
 namespace DB
 {
-
 Block QueryThreadLogElement::createBlock()
 {
-    return
-    {
-        {std::make_shared<DataTypeDate>(),          "event_date"},
-        {std::make_shared<DataTypeDateTime>(),      "event_time"},
-        {std::make_shared<DataTypeDateTime>(),      "query_start_time"},
-        {std::make_shared<DataTypeUInt64>(),        "query_duration_ms"},
-
-        {std::make_shared<DataTypeUInt64>(),        "read_rows"},
-        {std::make_shared<DataTypeUInt64>(),        "read_bytes"},
-        {std::make_shared<DataTypeUInt64>(),        "written_rows"},
-        {std::make_shared<DataTypeUInt64>(),        "written_bytes"},
-        {std::make_shared<DataTypeInt64>(),         "memory_usage"},
-        {std::make_shared<DataTypeInt64>(),         "peak_memory_usage"},
-
-        {std::make_shared<DataTypeString>(),        "thread_name"},
-        {std::make_shared<DataTypeUInt64>(),        "thread_id"},
-        {std::make_shared<DataTypeUInt64>(),        "master_thread_id"},
-        {std::make_shared<DataTypeString>(),        "query"},
-
-        {std::make_shared<DataTypeUInt8>(),         "is_initial_query"},
-        {std::make_shared<DataTypeString>(),        "user"},
-        {std::make_shared<DataTypeString>(),        "query_id"},
-        {DataTypeFactory::instance().get("IPv6"),   "address"},
-        {std::make_shared<DataTypeUInt16>(),        "port"},
-        {std::make_shared<DataTypeString>(),        "initial_user"},
-        {std::make_shared<DataTypeString>(),        "initial_query_id"},
-        {DataTypeFactory::instance().get("IPv6"),   "initial_address"},
-        {std::make_shared<DataTypeUInt16>(),        "initial_port"},
-        {std::make_shared<DataTypeUInt8>(),         "interface"},
-        {std::make_shared<DataTypeString>(),        "os_user"},
-        {std::make_shared<DataTypeString>(),        "client_hostname"},
-        {std::make_shared<DataTypeString>(),        "client_name"},
-        {std::make_shared<DataTypeUInt32>(),        "client_revision"},
-        {std::make_shared<DataTypeUInt32>(),        "client_version_major"},
-        {std::make_shared<DataTypeUInt32>(),        "client_version_minor"},
-        {std::make_shared<DataTypeUInt32>(),        "client_version_patch"},
-        {std::make_shared<DataTypeUInt8>(),         "http_method"},
-        {std::make_shared<DataTypeString>(),        "http_user_agent"},
-        {std::make_shared<DataTypeString>(),        "quota_key"},
-
-        {std::make_shared<DataTypeUInt32>(),        "revision"},
+    return {
+        {std::make_shared<DataTypeDate>(), "event_date"},
+        {std::make_shared<DataTypeDateTime>(), "event_time"},
+        {std::make_shared<DataTypeDateTime>(), "query_start_time"},
```

Comment:
Why did you reformat this code?

Reply:
I did a general formatting of the file, will keep in mind to not format blocks like this in future.